api/src/main/java/org/openmrs/module/attachments/AttachmentsConstants.java

```diff
@@ -32,8 +32,12 @@
 	
 	public static final String UPLOAD_ATTACHMENT_URL = "/rest/v1/" + ATTACHMENT_URI;
 	
+	public static final String OLD_UPLOAD_ATTACHMENT_URL = MODULE_BASE_URL + "/upload";
```

Comment:
This will be solved through ATT-4 and the constant will be named `LEGACY_UPLOAD_ATTACHMENT_URL `.